source/pom.xml

```diff
@@ -100,6 +106,14 @@
       <plugin>
         <groupId>org.jacoco</groupId>
         <artifactId>jacoco-maven-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>prepare-agent</id>
+            <goals>
+              <goal>prepare-agent</goal>
+            </goals>
+          </execution>
+        </executions>
```

Comment:
Think this is specified in the root pom in `pluginManagement` and as such doesn't need to be included? Could be wrong though, didn't test it.

Reply:
@Hilbrand did it for Register as well, so I thought if the master did.. but you are right I think.. Let me revert it and test it out.